docs/UsingLayering.md

```diff
@@ -0,0 +1,165 @@
+# Using OCP CoreOS Layering Phase 0
+
+In service to our [OCP CoreOS Layering Enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/ocp-coreos-layering/ocp-coreos-layering.md#phase-0).
+
+## Layering
+
+Layering lets you "layer" additional content on top of a Base OS Image using "containerfile" syntax and apply it to an OpenShift cluster using the MCO.
+
+As of 4.12:
+
+- The MCO uses the `rhel-coreos-8` native format base OS image by default instead of `machine-os-content`
```

Comment:
Linking to https://coreos.github.io/rpm-ostree/container/  would be useful for folks who wouldn't know what does native container mean.